hedyorg/hedy
# [FIX] Fix loading teacher adventures

This PR fixes two issues around teacher adventure customizations:

- Class customizations had the `teacher_adventures` field, which indicate what adventures are available in a particular class. This list of adventures had been replaced with the `adventure_order` set, and the UI that would fill the `teacher_adventures` field has been removed, so the field wasn't being set anymore. However, the `load_customized_adventures` function was still assuming that it only needed to load teacher adventures from the `teacher_adventures` field (and hence, since that field would always be empty, no adventures would be loaded).

- Second, on the class customization page, adventures wouldn't show up in the "available adventures" dropdown right away. That's because the code that would add options to the dropdown wasn't being called on page load, but only when the user twiddled the other parts of the user interface.

This PR addresses both of these issues.

**How to test**

Create an adventure, create a class, add a student to the class, add an adventure to the class, log in as the student, observe the adventure.
